app/uk/gov/hmrc/merchandiseinbaggage/views/PurchaseDetailsExportView.scala.html

```diff
@@ -17,19 +17,15 @@
 @import uk.gov.hmrc.merchandiseinbaggage.views.html.{Layout, components}
 @import uk.gov.hmrc.merchandiseinbaggage.controllers.routes._
 @import uk.gov.hmrc.merchandiseinbaggage.model.core.PurchaseDetailsInput
-@import uk.gov.hmrc.hmrcfrontend.views.html.components.HmrcCurrencyInput
 @import uk.gov.hmrc.merchandiseinbaggage.model.api.DeclarationType.Export
-@import uk.gov.hmrc.hmrcfrontend.views.Aliases.CurrencyInput
 @import uk.gov.hmrc.merchandiseinbaggage.controllers.DeclarationGoodsRequest
 
 @this(
-layout: Layout,
-formHelper: FormWithCSRF,
-h1: components.h1,
-errorSummary: components.errorSummary,
-text: components.inputText,
-currencyInput: HmrcCurrencyInput,
-button: components.button
+  govukInput: GovukInput,
+  layout: Layout,
+  formHelper: FormWithCSRF,
+  errorSummary: components.errorSummary,
+  button: components.button
```

Comment:
This was cleaned up. Also, CurrencyInput & HmrcCurrencyInput are deprecated see [9.9.0](https://github.com/hmrc/play-frontend-hmrc/releases/tag/v9.9.0). So the pound sign is not bold anymore as per [here](https://design.tax.service.gov.uk/hmrc-design-patterns/currency-input/). Now uses Input / GovukInput with prefix of £.